iotdb/src/test/java/org/apache/iotdb/db/engine/bufferwrite/BufferWriteProcessorNewTest.java

```diff
@@ -123,13 +126,18 @@ public void testWriteAndFlush()
       assertEquals(num, timeValuePair.getValue().getInt());
     }
     assertEquals(false, bufferwrite.isFlush());
+    long lastFlushTime = bufferwrite.getLastFlushTime();
     // flush asynchronously
     bufferwrite.flush();
-    assertEquals(true, bufferwrite.isFlush());
+    assertEquals(true, bufferwrite.getLastFlushTime() != lastFlushTime);
```

Comment:
Please confirm that the flush process will last no less than 1 ms in ANY MACHINE, or this will be unstable.

Reply:
correct... So changing it to nanoTime?

Reply:
I have change it to nanoSecond